provision/testdata/with_overrides.inp.yaml

```diff
@@ -54,6 +54,13 @@ kube_config:
   use_openshift_security_context_constraints: true
   allow_kube_api_default_epg: true
   no_wait_for_service_ep_readiness: true
+  service_graph_endpoint_add_delay:
+      delay: 30
+      services:
+      - name: ingress-service
+        namespae: openshift-ingress
```

Comment:
typo (should be namespace), I know it does not matter since this is just a test but better to correct

Reply:
done